bash_completions_test.go

```diff
@@ -74,26 +76,26 @@ func TestBashCompletions(t *testing.T) {
 		Run:                    emptyRun,
 	}
 	rootCmd.Flags().IntP("introot", "i", -1, "help message for flag introot")
-	rootCmd.MarkFlagRequired("introot")
+	er(rootCmd.MarkFlagRequired("introot"))
 
 	// Filename.
 	rootCmd.Flags().String("filename", "", "Enter a filename")
-	rootCmd.MarkFlagFilename("filename", "json", "yaml", "yml")
+	er(rootCmd.MarkFlagFilename("filename", "json", "yaml", "yml"))
```

Comment:
So I know that these show up as part of `errcheck` linter, but maybe the correct thing to do here is actually assert that there is no error since these are all in tests. 
Currently we aren't using any assertion libraries, maybe we write a helper like this for now. WDYT?
```suggestion
	assertNoErr(t, rootCmd.MarkFlagFilename("filename", "json", "yaml", "yml"))
```

```
func assertNoErr(t *testing.T, e error) {
  if e != nil {
    t.Error(e)
  }
}
```